### ISSUE TITLE
How do you especify the security Rule you want to update ?

### ISSUE THREAD

**Issue by carlos-alarcon** [itskashi/aws-ip-updater#1]
As far as I can see in the code the only param for the script is the tag to get updated

I would like to know how do you specify which security Group you want to update

I am thinking of a evolution of the script which will receive: securityGroupId, tag and hostname
It will resolv the hostname to fetch the IP address that hostname resolves (instead of looking for my own IP always)
Then go over the securityGroup and modify IP the the rules with such tag with the appropriated IP.

I will try to merge this changes as soon as I get something done., but first I will need to get how do yo specify the security group in you script.

Regadrs

**Comment by carlos-alarcon** [comment]
If I am not mistaken, you just do not modify an specific  security group. Instead you just filter all the rules from all security groups with the given tag. I will just provide a way to tell the security group ID instead.

**Comment by itskashi** [comment]
Yes, exactly and that is what the intention was, actually you can filter by security group, but IMO that is not needed or I could not think of a use case, so the idea is to update the security group rules across all the groups. 

The reason is you need such automation not for 1 security group, you need this when you have lot of security groups and want to update your rule in those groups. So I ignored the groups and instead just filter all the rules with the tag. 

But if you have a use-case, you can definitely filter by security group too, but at the end of the day it is the rule that we need to update.
